frontend/plugins/frontline_ui/src/widgets/relations/modules/ticket/Tickets.tsx

```
@@ -65,11 +66,13 @@ export const TicketRelationWidget = ({
     skip: !conversationId,
   });
 
-  const onClick = () => {
+  useEffect(() => {
+    if (!conversationDetail?.integration?.channelId) return;
     setTicketCreateDefaultValues({
       channelId: conversationDetail?.integration.channelId || undefined,
     });
-  };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)